projects/default/controllers/sumo_supervisor/SumoSupervisor.py

```diff
@@ -26,8 +26,8 @@
 
 
 def rotation_from_yaw_pitch_roll(yaw, pitch, roll):
-    """Compute the rotation from the roll pitch yaw angles."""
-    rotation = [0, 1, 0, 0]
+    """Compute the rotation from the roll pitch yaw angles, i.e. conversion Euler to Axis-Angle."""
```

Comment:
yaw pitch roll are not exactly the same as Euler angles, see discussion here: https://math.stackexchange.com/questions/147028/are-euler-angles-the-same-as-pitch-roll-and-yaw. Also I would fix the order to yaw pitch roll in the comment (order of the function arguments).

Reply:
yes I just wanted to give an indication on the formula used to compute the matrix, expect if you think it is cleared, else I can add something like: conversion Euler with XYZ representation to Axis-Angle.